server/website/website/migrations/0001_initial.py

```diff
@@ -97,6 +97,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(max_length=128)),
                 ('vartype', models.IntegerField(choices=[(1, 'STRING'), (2, 'INTEGER'), (3, 'REAL'), (4, 'BOOL'), (5, 'ENUM'), (6, 'TIMESTAMP')])),
+                ('default', models.CharField(max_length=32, null=True)),
```

Comment:
You should create a new migration file here.

Reply:
If I create a new file after 0003, eg. 0012, 0003 reports error when it puts data into the metric_catalog from fixture files.